src/mutator.cc

```diff
@@ -470,9 +470,9 @@ void Mutator::RegisterPostProcessor(const protobuf::Descriptor* desc,
 void Mutator::ApplyPostProcessing(Message* message) {
   const Descriptor* descriptor = message->GetDescriptor();
 
-  auto it = post_processors_.find(descriptor);
-  if (it != post_processors_.end()) {
-    it->second(message, random_());
+  auto it = post_processors_.equal_range(descriptor);
+  for (auto callback = it.first; callback != it.second; ++callback) {
```

Comment:
TODO: Instead of iterating through all registered callbacks, pick one (pseudo) randomly?